src/utils.js

```diff
@@ -25,10 +25,14 @@ const loadCSV = () => {
     const newValue = line.split(',')[1];
     result[oldValue] = newValue;
   }
+
+  // renderscript must be added to the canonical androidx-class-mapping.csv - it is not upstream
+  result['android.support.v8.renderscript'] = 'android.renderscript';
```

Comment:
I think it's better to add this in the `.csv` file

Reply:
I gather that since it is how you had it implemented

It seems like a small thing and admittedly it’s pretty trivial in the code - but

for critical infrastructure (which this will be as soon as RN0.60 comes out) I want to allow for verification if possible so I want the package to tolerate a pure download of the upstream CSV

I would like to implement a flag even that causes the package to do exactly that - download the csv at runtime from google - and include an SHA256 for the file we package so people can verify if they like

Reply:
I didn’t seem to do it correctly on the first cut anyway, and on one of my test rigs it turned a seemingly random AndroidManifest.xml to nothing but a hash of ‘undefined’’s so I’m not sure what’s going on at the moment but I’m working actively on it to get to 100% reproducible success

Reply:
I agree, removing the CSV file from the package and downloading it at runtime is a good idea! 
Adding the missing import to the `result` object is fine for now, but I think it should be handled differently in case we want to add more imports in the future

Reply:
yeah - could have a separate mapping file but from a premature optimization standpoint, renderscript was the only addition I’ve heard of and if there’s only 1, we can keep it simple. I’m still struggling to get it to work so that’s my priority - what node version are you using? I’ve got it pegged to 11 on CI + linux, but I can reproducibly hash AndroidManifest.xml files with node 12 + macOS for some reason? Really weird results - but I’m *positive* it can work so it must be something small I'm missing

Reply:
> [I didn’t seem to do it correctly on the first cut anyway](https://travis-ci.com/mikehardy/jetifier/builds/116689371), and on one of my test rigs it turned a seemingly random AndroidManifest.xml to nothing but a hash of ‘undefined’’s so I’m not sure what’s going on at the moment but I’m working actively on it to get to 100% reproducible success

the CSV file should not contain any empty lines

Reply:
Hmm - not sure that's it but I have to go to a meeting and won’t be back at it for a couple hours. I’m very persistent though :-), I’ll make it work - it (running via node) is obviously the right direction and I’ve already seen it correctly do everything but renderscript in CI so it’s just a matter of getting all the other combos too. it’ll take me time to test through all the combos but I don’t anticipate any code changes really, mostly just docs. I think this repo will end up about 80% documentation 20% code at this point 😆